### PR TITLE
adding post mapping of hello world

### PR DESCRIPTION
**Pull request by aditinikam**

### CODE REVIEW

File: src/main/java/com/xpi/xpiserver/controller/HelloController.java [hunk 1]
```diff
@@ -16,4 +19,9 @@ public class HelloController {
     public String hello() {
         return "Hello";
     }
+
+    @PostMapping("/hello")
+    public Map<String, String> hellos(final @RequestBody Map<String, String> input) {
```
Comment (varunavis) [inline src/main/java/com/xpi/xpiserver/controller/HelloController.java:24]: why are params of type map? shouldn' it be just string?

Comment (aditinikam) [reply src/main/java/com/xpi/xpiserver/controller/HelloController.java:24]: Can be but here it doesn't matter much what we use,
this is for testing here we can also check for key, value pair as well

Comment (varunavis) [reply src/main/java/com/xpi/xpiserver/controller/HelloController.java:24]: but theres 3 diff inputs, its not required

Comment (aditinikam) [reply src/main/java/com/xpi/xpiserver/controller/HelloController.java:24]: Which 3 inputs?

**Review by avivijay19 — APPROVED** [review]

**Review by avivijay19 — APPROVED** [review]
Good Work!!

**Pull request merged**